src/seedsigner/gui/screens/tools_screens.py

```diff
@@ -171,6 +171,73 @@ def update_title(self) -> bool:
 
 
 
+@dataclass
+class ToolsCoinEntropyEntryScreen(KeyboardScreen):
+    def __post_init__(self):
+        # Override values set by the parent class
+        # TRANSLATOR_NOTE: Inserts the number of flips required
+        self.title = _("Coin Flip 1/{}").format(self.return_after_n_chars)
+
+        # Specify the keys in the keyboard
+        self.rows = 2
+        self.cols = 2
+        self.key_height = GUIConstants.get_top_nav_title_font_size() + GUIConstants.EDGE_PADDING/4 + 2*GUIConstants.EDGE_PADDING
+        self.keys_charset = "".join([
+            "H",
+            "T",
+        ])
+
+        # Map Key display chars to actual output values
+        self.keys_to_values = {
+            "H": "1",
+            "T": "0",
+        }
+
+        # Now initialize the parent class
+        super().__post_init__()
+
+        self.components.append(TextArea(
+            text=_("(H)eads: 1   /   (T)ails: 0"),
+            screen_y = self.keyboard.rect[3] + 3*GUIConstants.COMPONENT_PADDING,
+        ))
+
+    def update_title(self) -> bool:
+        self.title = _("Coin Flip {}/{}").format(self.cursor_position + 1, self.return_after_n_chars )
+        return True
+
+
+@dataclass
+class ToolsCoinEntropySetwiseEntryScreen(KeyboardScreen):
+    current_set: int = 1
+    total_sets: int = 12
+    required_bits: int = 11
+
+    def __post_init__(self):
+        self.title = _(f"Set {self.current_set}/{self.total_sets}")
```

Comment:
see: https://github.com/SeedSigner/seedsigner/pull/720#discussion_r2037585915

tldr: we wouldn't want translators to have to write `self.current_set`; they should never have to worry about python syntax other than the variable brackets / named variables (which is already less than ideal but can't be avoided).